drafts/Stormless3.md

```diff
@@ -0,0 +1,322 @@
+
+
+> PRO TIP: I made workflow sequential to demonstrate data passing. It is more proper to run all three things in parallel: it is faster and more resilient as a failure of one integration point will not stop workflow execution and the others will still be invoked. Go ahead change the StepFunctions to parallel.
+
+That is it. Time to try. Deploy, invoke, check the logs. 
+
+
+You CURL funs know what to do with the new API Gateway endpoint for our StepFunction. where exposed our I'll show off again with httpie, you CURL fanatics know 
+
+```
+# DON'T COPY! Use YOUR ENDPOINT!
+
+http POST https://YOUR.ENDPOINT.amazonaws.com/dev/signup \
+body:='{"email":"santa@mad.russian.xmas.com", "first_name":"Santa", "last_name": "Claus"}'
+```
+
+Ok, httpie or curl, it returns the StepFunction execution ARN so that we can check on it and see how the StepFunction is being executed. How? I afraid gotta open a browser and login to your AWS Console. First, try to use AWS CLI, don't say I didn't show you how:
+
+```
+aws stepfunctions describe-execution --execution-arn arn:aws:states:us-east-1:00000000000:execution:SignupStepFunctionsStateMac-seo9CrijATLU:cbeda709-e530-11e7-86d3-49cbe4261318 --output json
+{
+    "status": "FAILED", 
+    "startDate": 1513738340.18, 
+    "name": "cbeda709-e530-11e7-86d3-49cbe4261318", 
+    "executionArn": "arn:aws:states:us-east-1:00000000000:execution:SignupStepFunctionsStateMac-seo9CrijATLU:cbeda709-e530-11e7-86d3-49cbe4261318", 
+    "stateMachineArn": "arn:aws:states:us-east-1:00000000000:stateMachine:SignupStepFunctionsStateMac-seo9CrijATLU", 
+    "stopDate": 1513738370.481, 
+    "input": "{\"body\":{\"email\":\"santa@mad.russian.xmas.com\",\"first_name\":\"Santa\",\"last_name\":\"Claus\"}}"
+}
+```
+
+
+This is an output for the execution that failed because RecordAC function timed out. The only valuable info here is `FAILED`. No kidding. I must say AWS don't give StepFunction the love it deserves. Not in CLI. If you think I missed some, check [the docs](http://docs.aws.amazon.com/cli/latest/reference/stepfunctions/index.html) yourself. 
+
+> PRO TIP: For debugging, invoke the StepFunction with `sls infoke stepf`: it creates the execution, waits for the completion, and prints the output to the terminal. Three AWS CLI commands in one. 
+> 
```

Comment:
You may not have wanted to include the `>` chars here. I'm guessing this carried over from the previous quote in your editor. This will likely break the formatting for this code snippet.